docs/userGuide/markBindInTheProjectWorkflow.md

```diff
@@ -36,5 +36,29 @@ You can keep the user docs in a separate directory (say `user-docs`) and set up
 Similarly, you can keep the dev docs in a separate directory (sey `dev-docs`) and set up Netlify to deploy the site when there is an update to the `master` branch; that way, developers can see the latest version of dev-docs via the Netlify site.
 </div>
 
+#### Converting existing project documentation/wiki
+
+MarkBind supports the automatic conversion of an existing GitHub wiki or `docs` folder containing Markdown files.
+
+A MarkBind conversion involves the following:
+- Adding a Home page: If your project already has a `README.md` or `Home.md`, the content will be copied over to `index.md`. Otherwise, a default home page will be added.
+- Adding an About Us page: If your project already an `About` page, the content will be copied over to `about.md`. Otherwise, a default about page will be added.
```

Comment:
> - Adding an About Us page: If your project already an `About` page, ...

Do our code actually check for this?

Reply:
Yes, the conversion process checks for `about.md` and uses it if present.

Reply:
I should word it differently: `If your project already has an about.md, it will be used as the About page. ...`